extensions/cornerstone/src/synchronizers/frameViewSynchronizer.ts

```diff
@@ -43,7 +43,8 @@ const createFrameViewSynchronizer = (synchronizerName: string): Synchronizer =>
   const synchronizer = SynchronizerManager.createSynchronizer(
     synchronizerName,
     EVENTS.STACK_VIEWPORT_SCROLL,
-    frameViewSyncCallback
+    frameViewSyncCallback,
+    { auxiliaryEvents: [{ name: EVENTS.CAMERA_MODIFIED, source: 'element' }] }
```

Comment:
I kinda messed up on a previous PR. The camera modified event is still needed when the HP is initially opened.